main/u-boot-rock64-rk3328/update.py

```diff
@@ -0,0 +1,2 @@
+pkgname = "u-boot"
+ignore = ["2023.07.*"]
```

Comment:
`pkgver` higher than only `ignore` pattern, should be dropped

Reply:
Done, but same should go for https://github.com/chimera-linux/cports/blob/86a1094b94bba8f44f7732f8b01c20d8f9049ab2/main/u-boot-rockpro64-rk3399/update.py#L2 then, right?